src/app/components/navigation/navigation.component.scss

```diff
@@ -1,9 +1,6 @@
-#pa_header_pqk {
+.pa_header {
```

Comment:
prefer `-` over `_` in css class names
If you change this, you can merge without requesting another review